pages/fr.tsx

```diff
@@ -0,0 +1,13 @@
+import Home from "~/screens/home";
+
+export default function Fr() {
+  return <Home />;
+}
+
+export function getStaticProps() {
+  return {
+    props: {
+      locale: "fr",
```

Comment:
Right now, accessing `/en` returns a 404. Having everything on a single page really makes things easier for us in this case. Abusing this, could it be possible to have a catch-all route to define future locales? We can redirect to the default one or throw a 404 for invalid routes. It would look like `pages/[locale].tsx`, and we could retrieve the locale from `router.query.locale`.

The [`app` directory](https://beta.nextjs.org/docs/routing/fundamentals#the-app-directory) could potentially fix the static generation issue, but I didn't use it and don't know if it's possible yet.

Reply:
Thinking about it, we could `getStaticPaths` valid locales and disable fallback

Reply:
That's something I was thinking about doing before, but then decided to keep things simple. The new `app` directory doesn't currently support `next export` yet afaik, otherwise that would've been interesting to use.